client/landing/gutenboarding/components/login-form/index.tsx

```diff
@@ -59,18 +85,32 @@ const LoginForm = () => {
 	// todo: may need to be updated as more states are handled
 	const shouldShowPasswordField = loginFlowState === 'ENTER_PASSWORD';
 
+	const changeEmailOrUsernameLinkText = includes( usernameOrEmailVal, '@' )
+		? NO__( 'Change Email Address' )
+		: NO__( 'Change Username' );
+
 	return (
 		<Modal
 			className="login-form"
 			isDismissible={ false }
 			title={ NO__( 'Log in to save your changes' ) }
-			onRequestClose={ handleClose }
+			onRequestClose={ closeModal }
 		>
 			<form onSubmit={ handleLogin }>
+				{ shouldShowPasswordField && (
+					<Button
+						className="login-form__change-username"
+						isLink={ true }
+						onClick={ changeEmailAddress }
+					>
+						<Icon icon="arrow-left-alt" size={ 18 } />
+						{ changeEmailOrUsernameLinkText }
+					</Button>
+				) }
 				<TextControl
-					label={ NO__( 'Email Address or Username' ) }
+					label={ ! shouldShowPasswordField ? NO__( 'Email Address or Username' ) : '' }
 					value={ usernameOrEmailVal }
-					// disabled={ isLoading }
+					disabled={ shouldShowPasswordField }
 					onChange={ setUsernameOrEmailVal }
 					placeholder={ NO_x(
 						'E.g., yourname@email.com',
```

Comment:
Maybe we don't need a placeholder since the user can enter either a username or an email (?)

Reply:
+1, I'd stick to how it's in production since I'm sure it was considered:

<img width="518" alt="image" src="https://user-images.githubusercontent.com/87168/76403633-db5c0180-638d-11ea-84b2-486d9dc6d11b.png">

Reply:
Aside, if you decide to move on with email placeholder, we have internal example domains we can use. Examples across Calypso. :-)

Reply:
ok, yep I'll take the placeholder out and make it the same as the existing login page 👍